update_python_spec.lisp

```diff
@@ -1,13 +1,15 @@
 #!/usr/bin/env sbcl --script
 (load "~/.sbclrc")
 (ql:quickload :rpcq)
+(ql:quickload :split-sequence)
```

Comment:
It doesn't really matter, but do you still want this?

Reply:
removed